test/apps/iodemo/run_io_demo.sh

```diff
@@ -198,7 +212,7 @@ check_params()
 collect_ip_addrs()
 {
 	# convert the output of 'ip' to 'host:ip' list
-	host_ips=$(pdsh -w ${host_list} ip -4 -o address show ${net_if} |
+	host_ips=$(${launcher} ${host_list} ip -4 -o address show ${net_if} |
```

Comment:
i dont think it is correct regex, quick check got this:

```
$pdsh -w $(hostname),$(hostname) "ip -4 -o address show eth0" | sed -e 's/^\(\w*\): .* inet \([0-9\.]*\).*$/\1:\2/g'
mtr-vdi-326: 2: eth0    inet 10.209.100.58/22 brd 10.209.103.255 scope global dynamic eth0\       valid_lft 537489sec preferred_lft 537489sec
mtr-vdi-326: 2: eth0    inet 10.209.100.58/22 brd 10.209.103.255 scope global dynamic eth0\       valid_lft 537489sec preferred_lft 537489sec
```

Reply:
maybe smth like this: ?
```
$pdsh -b -w  $(hostname),$(hostname) "ip -4 -o address show eth0  | sed -Ene 's/.*inet ([0-9.]+).*/\1/p'"
mtr-vdi-326: 10.209.100.58
mtr-vdi-326: 10.209.100.58
```

Reply:
fixed

Reply:
also, in line 219:
it will not return host_ips as ```host IP``` but as separate tokens, i.e.
```
host
ip
host
ip
```

Reply:
it will be "host1:ip1 host2:ip2" (e.g token for each host)

Reply:
your fix works fine! thanks